Code/Tools/AssetProcessor/native/ui/ProductAssetTreeModel.cpp

```diff
@@ -169,6 +171,23 @@ namespace AssetProcessor
         const AzToolsFramework::AssetDatabase::ProductDatabaseEntry& product,
         bool modelIsResetting)
     {
+        AZStd::string platform;
+        m_sharedDbConnection->QueryJobByProductID(
+            product.m_productID,
+            [&](AzToolsFramework::AssetDatabase::JobDatabaseEntry& jobEntry)
+            {
+                platform = jobEntry.m_platform;
+                return true;
+            });
+
+        // Common platform is currently only for Intermediate Product Assets,
```

Comment:
Have a concern on the naming (may not related to this PR though): Common platform makes it sound like the asset is shared by all the platforms (pc, linux, mobile, etc.) Will we use the same term when adding the feature for avoiding duplicate processing for assets that are shared by all the platforms?

Reply:
With Mike's suggested change, I'm not checking for common platform anywhere anymore, I'm checking the product flag